Chummer/customdata/German Data Changes/manifest.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<manifest>
+  <name>German Data Changes</name>
+  <guid>1ea1a21e-7cca-4ce9-b062-8f306880e8a6</guid>
+  <!--version and author are just here to serve as an example-->
+  <version>1.0</version>
+  <author>Multiple</author>
```

Comment:
Hmm, maybe an `<authors>` node that can list any number of `<author>` children instead? The first one could be assumed to be the primary, or you could also allow a different one to be manually set as the primary author via attributes.

Reply:
Good idea. Marking the main author with an attribute and defaulting to the first one in the list would be the way to go.